server/datastore/mysql/microsoft_mdm_test.go

```diff
@@ -1865,22 +1880,39 @@ func testSetOrReplaceMDMWindowsConfigProfile(t *testing.T, ds *Datastore) {
 	cp1 := *windowsConfigProfileForTest(t, "N1", "N1")
 	err := ds.SetOrUpdateMDMWindowsConfigProfile(ctx, cp1)
 	require.NoError(t, err)
+	profNoTmN1 := getProfileByTeamAndName(nil, "N1")
 
 	// creating the same profile for Apple / no-team fails
 	_, err = ds.NewMDMAppleConfigProfile(ctx, *generateCP("N1", "I1", 0))
 	require.Error(t, err)
 
+	cp1.UploadedAt = profNoTmN1.UploadedAt
 	profs1 := expectWindowsProfiles(t, ds, nil, []*fleet.MDMWindowsConfigProfile{&cp1})
 
-	// update the profile for no-team
+	// wait a second to ensure timestamps in the DB change
+	time.Sleep(time.Second)
```

Comment:
To be honest, I think it would make a difference mostly for tests, but yeah that'd still be nice. I hate adding sleeps in tests, at least then they could be almost unnoticeable.